src/styles/Input.scss

```diff
@@ -39,8 +33,53 @@
             outline-style: solid;
         }
 
+        &:disabled{
+            border-radius: 0rem !important;
+            border: 1px #BBBBBB;
+            border-style: solid;
+            fill: #EEEEEE;
+        }
     }
 
+    ::placeholder{
+        font-weight: 400;
+        text-transform: none;
+        font-size: 14px;
+        color: $LIGHT-GREY;
+    }
+}
+
+.clr-subtext{
+    display: block;
+    font-size: 12px;
+    line-height: 2rem;
+    margin-top: .3rem;
+    fill: #666666;
+    font-weight: 400;
```

Comment:
as suggested by Mahesh please create constants for common styles such as ,
font-weight: 400;
letter-spacing: 0px;
line-height: 20px;